data/companies.json

```diff
@@ -736,6 +736,22 @@
         "twitter": "https://twitter.com/yarsalabs/",
         "youtube": "https://www.youtube.com/channel/UCqJAqxSBEmpdCqwgQG3Qoeg"
       }
-    }
+    },
+    {
+      "name": "YCo Pvt. Ltd.",
+      "establishedAt": 871582500,
+      "location": "Lokeshwor Tole, Manbhawan, Laltipur, Nepal",
+      "website": "https://yco.com.np/",
+      "email": "info@yco.com.np",
+      "phone": "977 (1) 5551110, 2113000",
```

Comment:
The phone number format is `+977-15551110` and please only add 1 number of the company

Reply:
okii